examples/wasm-example/src/services.rs

```diff
@@ -143,7 +143,7 @@ pub async fn extension_signature_for_partial_extrinsic(
     let spec_version = encode_to_hex_reverse(&api.runtime_version().spec_version);
     let transaction_version = encode_to_hex_reverse(&api.runtime_version().transaction_version);
     let mortality_checkpoint = encode_to_hex(&api.genesis_hash());
-    let era = encode_to_hex(&subxt::config::extrinsic_params::Era::Immortal);
+    let era = encode_to_hex(&0u8);
```

Comment:
I guess `0` is `Era::Immortal` would be nice with a comment to explain that, this is just an example right?

Reply:
Yup just an example; I'll add a comment :)